cmd/agent/main_windows.go

```diff
@@ -38,8 +39,5 @@ func main() {
 	}
 	defer log.Flush()
 
-	if err := command.MakeCommand(subcommands.AgentSubcommands()).Execute(); err != nil {
-		fmt.Println(err)
-		os.Exit(-1)
-	}
+	runcmd.Run(command.MakeCommand(subcommands.AgentSubcommands()))
```

Comment:
I think we may want to do a bit of renaming in this PR at least for the new module. Reducing stutter in the daisy chained line like this could go a long way in improving the ergonomic of the new additions. Maybe something like ` runner.Run(...)`, `runner.Execute(...)`, `cmdwrapper.Invoke(...)`, etc or some permutation but I'm open to alternatives.

Reply:
Can you be more specific in what you're suggesting?  These need to be three different packages due to dependencies, but the function names could change.

Reply:
I think my comment at least for this PR scope would be to leave the package structure as-is and just to specifically  improve the `runcmd` and `Run` names to reduce stutter in both isolation and when used in a longer combination like we do in this line.